### DISCUSSION TITLE
Will this work for macOS Catalina?

### DISCUSSION THREAD

**Discussion by ddimitrov777** [brock/node-reinstall#51] [Q&A] [answered]
I saw in a comment elsewhere that there may be an issue because newer OS uses zsh?

Also, what examples of things that would be deleted from those directories? Would it be like e.g. anything that has ever been installed with brew?

**Comment by brock** [comment, marked as answer]
I am on Big Sur and it works for me. It worked before I upgraded, so I'm not aware of any issues. The directories that were included have been used by several installers over the years, including brew, the downloadable installer from the Node website, and several others that people recommended in the original Stack Overflow article.
